### PR TITLE
Proposed COVID-19 vaccination policy for PDX 2021

### PR DESCRIPTION
**Pull request by obfuscurity**
Putting this out there for review and discussion. The goal here is to set expectations for anyone participating in the event.

I will require this for our staff and vendors, but it's very unlikely that we could _effectively_ enforce this policy for everyone. Vaccination cards are non-standardized and easily forged.

Regardless, it's my expectation that folks _will_ abide by this policy, because this event attracts intelligent and conscientious individuals who understand the risks and are doing their part to combat this pandemic. 🥰

### CODE REVIEW

**Review by owen-d — APPROVED** [review]
Thank you

**Comment by eapotapov** [comment]
@obfuscurity might make sense to mention if there are any requirements for vaccine types.
I'm vaccinated by Russian Sputnik-V, it's legal for me to enter the states and it's approved by some countries, but for instance most EU countries wait for EU universal approval. Also there are some vaccines from China etc.

**Pull request merged**
